data/fields/hazard_highway.json

```diff
@@ -0,0 +1,31 @@
+{
+    "key": "hazard",
+    "type": "combo",
+    "strings": {
+        "options": {
+            "animal_crossing": "Animal Crossing",
+            "bump": "Bump",
+            "children": "Children",
+            "curve": "Curve",
+            "curves": "Multiple Curves",
+            "cyclists": "Cyclists",
+            "dangerous_junction": "Dangerous Junction",
+            "dip": "Dip",
+            "falling_rocks": "Falling Rocks",
+            "frost_heave": "Frost Heave",
+            "horse_riders": "Horse Riders",
+            "ice": "Ice",
+            "landslide": "Landslide",
+            "loose_gravel": "Loose Gravel",
+            "low_flying_aircraft": "Low Flying Aircraft",
+            "pedestrians": "Pedestrians",
+            "queues_likely": "Queues Likely",
```

Comment:
“Queue” isn’t quite as understandable in the default locale, American English. A more effective label would be “Stopped Traffic”. (The signs in most of the U.S. say “Watch for Stopped Traffic”.)